modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/EditObjectField.tsx

```diff
@@ -129,7 +150,7 @@ export default function EditObjectField({
 		}
 	}, [errors]);
 
-	return (
+	return objectDefinition.externalReferenceCode ? (
```

Comment:
Hey @igor-franca, this will not be necessary as we will no longer have the objectDefinition state in this file, you can remove this conditional